### ISSUE TITLE
Can’t Re-Download Watch App for Tesla

### ISSUE THREAD

**Issue by pm41613** [Kemmey/Kemmey-TeslaWatch-Public#192]
Recently purchased this app.  Had to do a full reset of my Apple Watch.

I’ve attempted to go into the Watch App Store only to discover that I’m being requested to pay for the app again to download.

The app doesn’t appear in my purchased apps on my watch, though it does appear as such on my phone.

**Comment by Kemmey** [comment]
@pm41613 That's a bug on apples part. They don't properly show purchased apps on the watch App Store. When you tap to purchase again, it will say you already own it and download for free. If not, you're using different Apple ids now than previously. In that case, ask Apple for a refund for the new purchase.

**Comment by pm41613** [comment]
Got it. Thanks! That worked.

**Comment by tydaking** [comment]
Is this similar to the issue I'm having where I changed Apple watches and the app is still showing purchased, but the app isn't in the watch app to use on the new watch

**Comment by Kemmey** [comment]
@tydaking Yeah, just download again, but from the watch itself: https://testflight.apple.com/join/FdzFOLZv

**Comment by tydaking** [comment]
> @tydaking Yeah, just download again, but from the watch itself: https://testflight.apple.com/join/FdzFOLZv

Thanks for the link, that worked.

**Comment by Kemmey** [comment]
@tydaking Whoops, that was actually the wrong link :-D you just installed the beta version - which is cool, can always use more testers :-D 

What I meant to link to though, was instructions to download from store app on watch : https://github.com/Kemmey/Kemmey-TeslaWatch-Public/issues/5#issuecomment-658742571

**Comment by tydaking** [comment]
No prob at all, I don't mind testing. Thanks.
